assignments/02-python-access.md

```diff
@@ -0,0 +1,41 @@
+# Assignment 2: Practicing Python and Accessing Data
+
+
+## Objective & Evaluation
+
+This assignment is an opportunity to earn level 1 and 2 achievements in `python` and `access` and begin working toward level 1 in `summarize`.
+
+
+Accept the assignment on [GitHub Classroom](). It contains a notebook with some template structure (and will set you up for grading)
```

Comment:
Assignment not created, template here: https://github.com/rhodyprog4ds/accessing-data